base/applications/rapps/lang/ro-RO.rc

```diff
@@ -1,4 +1,13 @@
-/* Ștefan Fulea (stefan dot fulea at mail dot md) */
+/*
+ * PROJECT: ReactOS Applications Manager (RAPPS)
+ * FILE:    base/applications/rapps/lang/ro-RO.rc
+ * PURPOSE: Romanian Translation for RAPPS
+ * TRANSLATORS:
+ *          Ștefan Fulea (stefan dot fulea at mail dot md)
+ *          Bișoc George (fraizeraust99@gmail.com)
+ * HISTORY:
+ *          29/12/2017 -- Update by Fraizeraust alias Bișoc George (fraizeraust99@gmail.com)
```

Comment:
This is not necessary, that's what "git log" is for :)

Reply:
Must I delete the whole header or just the "HISTORY" parth, Thomas Faber?

Reply:
The rest of the header seems fine.